pkg/commands/git_commands/commit_loader.go

```diff
@@ -349,12 +349,25 @@ func (self *CommitLoader) setCommitMergedStatuses(refName string, commits []*mod
 	if ancestor == "" {
 		return commits
 	}
+	inMainBranch := false
+	branchRef, err := self.cmd.New(
+		NewGitCmd("symbolic-ref").Arg("HEAD").
```

Comment:
This is not quite right. It works for the commits panel, but the same code also runs for the commits of a branch if you press enter on it in the branches panel; in that case the branch isn't necessarily the same as the currently checked-out branch. But we still want the color-coding to work correctly in that case, too.

One way to solve this is to run the "symbolic-ref" command only if `refName == "HEAD"`, but set `branchRef` to `refName` otherwise.

Maybe @jesseduffield can think of a nicer way of doing this; ideally we wouldn't have to make a git call at all here, but pass the branch name in somehow.